python/tests/requirements.txt

```diff
@@ -0,0 +1,46 @@
+#
```

Comment:
Should the requirements.in file be included in the PR too?

Reply:
For some reason there was already a requirements.in file here (https://github.com/SatcherInstitute/health-equity-tracker/blob/master/python/tests/requirements.in) but not a requirements.txt